doc/source/whatsnew/v2.0.3.rst

```diff
@@ -21,6 +21,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
+- Bug in :class:`RelativeDeltaOffset` which caused inconsistent behavior upon multiplying a :class:`DateOffset` (:issue:`47953`)
```

Comment:
`RelativeDeltaOffset` isn't technically part of the pandas API - could this be phrased just in terms of `DateOffset`?

Reply:
Changed the phrasing. Please let me know if this works.

Reply:
looks good!